busted/modules/files/terra.lua

```diff
@@ -1,7 +1,7 @@
 local path = require 'pl.path'
 
 local ret = {}
-local ok, terralib = not not terralib, terralib --pcall(function() return require 'terralib' end)
+local ok, terralib = not not terralib, terralib
```

Comment:
sorry for coming back to this, but I think the proper approach here would be to drop the `ok` variable altogether.

```lua
local terralib = terralib
```

and [further down](https://github.com/Olivine-Labs/busted/pull/606/files/eac6528531a43f9d4b42cc69611ae25afd8f55ff..f0c68daed4a3a0bc65a76d1e0f188fe1b043d57d#diff-c58fa8a6ef8f7ec3172594aea382b534R13), replace `ok` with `terralib`

Reply:
I'm planning to make a code change to Terra that will make it work by `require 'terra'` consistently whether it is being loaded as a library from terra.so or being run by in the terra executable, so the line of code would change to `local ok, terralib = pcall(require, 'terra')` again. I was intending to leave the logic around ok intact so that it doesn't take as much to change back later. But there isn't much code, so I suppose it isn't a big deal to make that change. Actually, while writing this, I had an idea to make an improvement.